k8s/upgrades/0.9.0-1.0.0/README.md

```diff
@@ -0,0 +1,131 @@
+# UPGRADE FROM OPENEBS 0.9.0 TO 1.0.0
+
+## Overview
+
+This document describes the steps for upgrading OpenEBS from 0.9.0 to 1.0.0
+
+The upgrade of OpenEBS is a three step process:
+- *Step 1* - Prerequisites
+- *Step 2* - Upgrade the OpenEBS Operator
+- *Step 3* - Upgrade the OpenEBS Volumes from previous versions (0.9.0)
+
+#### Note: It is mandatory to make sure to that all OpenEBS control plane components and volumes are running with version 0.9.0 before the upgrade.
+
+### Terminology
+- *OpenEBS Operator : Refers to maya-apiserver & openebs-provisioner along w/ respective services, service a/c, roles, rolebindings*
+- *OpenEBS Volume: Storage Engine pods like cStor or Jiva controller(aka target) & replica pods*
+
+### Download the upgrade scripts
+
+The easiest way to get all the upgrade scripts is via git clone.
+
+```sh
+$ mkdir upgrade-openebs
+$ cd upgrade-openebs
+$ git clone https://github.com/openebs/openebs.git
+$ cd openebs/k8s/upgrades/0.9.0-1.0.0/
+```
+
+## Step 1: Prerequisites
+
+*All steps described in this document need to be performed on the Kubernetes master or from a machine that has access to Kubernetes master*
+
+ - Run below command to update OpenEBS control plane components labels.  
```

Comment:
Run below command to perform pre-upgrade operations on OpenEBS related components.